.gitignore

```diff
@@ -16,3 +16,4 @@ logs/
 # Ignore most CSVs, except those that are used as dbt seeds
 *.csv
 !dbt/seeds/**/*.csv
+venv/
```

Comment:
I've been using a root level venv to prototype python scripts and figured it would be good to ignore it.